### ISSUE TITLE
Heavy lag when typing in edit boxes

### ISSUE THREAD

**Issue by frostytoaster** [nvaccess/nvda#10785]
### Steps to reproduce:
1. Locate any edit box.
2. Ensure the speaking of typed character is turned on.
3. Begin typing.

### Actual behavior:
There is a fairly significant delay from when the key is pressed and when NVDA reads out the key. 
### Expected behavior:
NVDA should report the key press instantly.
#### NVDA installed/portable/running from source:
Installed.
#### NVDA version:
2019.3.1
#### Windows version:
Windows 10 Version	10.0.18362 Build 18362
 
#### Name and version of other software in use when reproducing the issue:
Lag appears reguardless of which edit box is being used.  Tested with Chrome edit boxes and address bar, latest version.  Tested with notepad, run dialog box, start menu and Jarte version 6.2 
#### Other information about your system:
Lag is present reguardless of speach synthasiser used.  Tested with Demo of elequence, Windows onecore voices and with NVDA's Espeak.
#### Does the issue still occur after restarting your PC?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
It didn't seem to be an issue in NVDA version 2019.3

**Comment by Adriani90** [comment]
Did you test also with all addons disabled?
I cannot reproduce this in NVDA 2019.3.1.
Also please restart your computer and try to reproduce it after that.

**Comment by frostytoaster** [comment]
Hi,
Is there a way to disable all addons? I can’t find it.  I am only coming across individual disable buttons for each addon.

**Comment by frostytoaster** [comment]
Hi,
So I attempted to run NVDA without addons but am still getting the same lag.

**Comment by Brian1Gaff** [comment]
I don't see this on Windows7, though sometimes in the file save type 
dialogue, if you attempt to go to the start often existing name and type the 
path, this can have a very long lag before what you type is spoken.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users

**Comment by Brian1Gaff** [comment]
To disable all add ons:
 use nvda and q go down to restart with add ons disabled.
 Cheers.
 Brian

bglists@blueyonder.co.uk 
Sent via blueyonder. 
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users

**Comment by Brian1Gaff** [comment]
Just a shot in the dark, have you been playing with any of the advanced 
options at all?
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users

**Comment by frostytoaster** [comment]
hi Brian, I have not touched the advanced settings. I don't know why it is 
doing this but it has made nvda very unusable.

**Comment by frostytoaster** [comment]
Hi again,
So I attempted an uninstall and reinstall and am still suffering with the same keyboard lag.  Any more suggestions would be grately appreciated as I am almost in a situation where by I will have to either use another screen reader or go back to an older version of NVDA.

**Comment by Brian1Gaff** [comment]
Does it go back to how it used to be if you  go to the older version again?

I think a few diagnostics are needed. If you set the logging to debug and 
look at the log when you type are there any indications of a problem. It 
used to be the old frozen in stack with a timeout, but I think its a bit 
different in Python 3.
 Have you run the dll re registration tool?



Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users

**Comment by frostytoaster** [comment]
Hi Brian,

I created a log file but am not  sure what to look for.  I ran the dll fixing option under tools but alas, no improvement there  either.  Here is the log file that was generated.
Thank you.

**Comment by frostytoaster** [comment]
Hi,
So I have tried everything I can think of and I still can't get it to work. Please help.  I have had to use other screen readers to keep up my productivity. I will attach my log file here just in case anyone else can spot the issue.
[nvda.log](https://github.com/nvaccess/nvda/files/4228117/nvda.log)

**Comment by frostytoaster** [comment]
Hello,
I finally found a solution to the issue.  I simply deleted the NVDA directory from %App data%.  So I still have no idea what caused it but at least I can use NVDA again.
